packages/flame/lib/src/components/position_component.dart

```diff
@@ -163,8 +163,17 @@ class PositionComponent extends Component {
   /// component as seen from the parent's perspective, and it is equal to
   /// [size] * [scale]. This is a computed property and cannot be
   /// modified by the user.
-  Vector2 get scaledSize =>
-      Vector2(width * scale.x.abs(), height * scale.y.abs());
+  Vector2 get scaledSize {
+    return Vector2(width * scale.x.abs(), height * scale.y.abs());
+  }
+
+  /// The resulting angle after all the ancestors and the components own angle
+  /// has been applied.
+  double get absoluteAngle {
```

Comment:
should we consider caching this somehow? might be more effort than its worth though

Reply:
I actually had caching first, but then considered it a premature optimization since this is only on primitive types, doesn't create any new objects and it is basically just going through a short linked list, I think it should be fine.